### PR TITLE
Data Generation

### PR DESCRIPTION
**Pull request by TimothyAkana**
Sorry about the git commit history. I accidentally committed a large CSV and had to rollback some of the git commits in order to get it back out of the history. This PR generally covers three main topics:
1. Data generation scripts that load into CSVs.
2. Files containing the Schemas and Table setups for Postgres and Cassandra.
3. Some files used to load the CSVs into the respective databases.

### CODE REVIEW

**Comment by TimothyAkana** [comment]
@shaileepatel Can you take a look at this PR related to data generation?
@rolosaavedra1 Tagging you too in case you wanted to take a look.

File: sdcSeeding/Cassandra/cassandraDataGeneration.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  let newDate = new Date(date);
+  const randomDays = Math.floor(Math.random() * 365);
+  newDate.setDate(newDate.getDate() + randomDays)
+  return newDate;
+}
+
+const zeroPadding = (number, totalNumDigits) => {
+  let string = number.toString();
+  while (string.length < totalNumDigits) {
+    string = '0' + string
```
Comment (shaileepatel) [inline sdcSeeding/Cassandra/cassandraDataGeneration.js:34]: if one of the item is a string, you wont require quotes for 0

File: sdcSeeding/Cassandra/cassandraDataGeneration.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    numRecords--;
+    const name = lorem.generateSentences(1);
+    const price = 10000 * (Math.floor(Math.random() * 250) + 50);
+    records.push({home_id: id, home_name: name, home_price: price})
+  }
+  return records;
+}
+const sharedHomeFields = generateSharedFields(10000000);
+
+const writeHomePricesStream = fs.createWriteStream('base_home_prices.csv');
```
Comment (shaileepatel) [inline sdcSeeding/Cassandra/cassandraDataGeneration.js:66]: Rolo mentioned a good point in mine about some repeated code. Noticed something similar in yours too, maybe you can refactor if you want

File: sdcSeeding/Cassandra/cassandraDataGeneration.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+      const price = current.home_price;
+      const data = `${home_id},${name},${price}\n`
+      if (numHomes === 0) {
+        //This is the last write, so call the callback afterwards
+        writer.write(data, encoding, callback);
+      } else {
+        ok = writer.write(data, encoding);
+      }
+    } while ((numHomes > 0) && ok);
+    if (numHomes > 0) {
```
Comment (shaileepatel) [inline sdcSeeding/Cassandra/cassandraDataGeneration.js:88]: like this piece of code

File: sdcSeeding/Cassandra/cassandraSchema.cql [hunk 4]
```diff
@@ -0,0 +1,51 @@
+CREATE KEYSPACE  IF NOT EXISTS affordability
```
Comment (shaileepatel) [inline sdcSeeding/Cassandra/cassandraSchema.cql:1]: dont know much about cassandra, but from what I understand looks good.

File: sdcSeeding/Postgresql/postgresqlDataGeneration.js [hunk 5]
```diff
@@ -0,0 +1,165 @@
+const fs = require('fs');
```
Comment (shaileepatel) [inline sdcSeeding/Postgresql/postgresqlDataGeneration.js:1]: similar comments from Cassandra seeding

**Review by shaileepatel — APPROVED** [review]

File: sdcSeeding/Cassandra/cassandraDataGeneration.js (hunk 1 above)
Comment (TimothyAkana) [reply sdcSeeding/Cassandra/cassandraDataGeneration.js:34]: Good call!

**Pull request merged**